cmd/erasure-sets.go

```diff
@@ -285,7 +287,7 @@ func (s *erasureSets) monitorAndConnectEndpoints(ctx context.Context, monitorInt
 	time.Sleep(time.Duration(r.Float64() * float64(time.Second)))
 
 	// Pre-emptively connect the disks if possible.
-	s.connectDisks()
+	s.connectDisks(false)
```

Comment:
What's the reason behind not printing error logs in the first attempt to connect to drives?

Reply:
Unnecessary since servers are coming up. We log the same information elsewhere it's just frivolous here.

It gets printed anyway after the next loop, if they are still not online. Just that we don't have to print the error at once.